Packs/MicrosoftGraphDeviceManagement/Integrations/MicrosoftGraphDeviceManagement/MicrosoftGraphDeviceManagement.py

```diff
@@ -0,0 +1,462 @@
+            'CodeIntegrityPolicy': device_health_attestation_state.get('codeIntegrityPolicy'),
+            'BootRevisionListInfo': device_health_attestation_state.get('bootRevisionListInfo'),
+            'OperatingSystemRevListInfo': device_health_attestation_state.get('operatingSystemRevListInfo'),
+            'HealthStatusMismatchInfo': device_health_attestation_state.get('healthStatusMismatchInfo'),
+            'HealthAttestationSupportedStatus': device_health_attestation_state.get('healthAttestationSupportedStatus')
+        }),
+        'SubscriberCarrier': raw_device.get('subscriberCarrier'),
+        'MEID': raw_device.get('meid'),
+        'TotalStorageSpaceInBytes': raw_device.get('totalStorageSpaceInBytes'),
+        'FreeStorageSpaceInBytes': raw_device.get('freeStorageSpaceInBytes'),
+        'ManagedDeviceName': raw_device.get('managedDeviceName'),
+        'PartnerReportedThreatState': raw_device.get('partnerReportedThreatState')
+    })
+
+
+def build_device_human_readable(device: dict) -> dict:
+    """
+    Builds a device human readable object
+    :param device: The raw device object
+    :return: The device human readable object
+    """
+    device_human_readable: dict = dict()
+    for header in HEADERS['raw_device']:
+        index: int = HEADERS['raw_device'].index(header)
+        device_human_readable[HEADERS['device'][index]] = device.get(header)
+    return assign_params(**device_human_readable)
+
+
+''' COMMANDS '''
+
+
+def list_managed_devices_command(client: MsGraphClient, args: dict) -> None:
+    limit: int = try_parse_integer(args.get('limit', 10), err_msg='This value for limit must be an integer.')
+    list_raw_devices, raw_response = client.list_managed_devices(limit)
+    list_devices: list = [build_device_object(device) for device in list_raw_devices if device]
+    list_devices_hr: list = [build_device_human_readable(device) for device in list_raw_devices if device]
+    entry_context: dict = {'MSGraphDeviceManagement.Device(val.ID === obj.ID)': list_devices}
+    human_readable: str = 'No managed devices found.'
+    if list_devices:
+        human_readable = tableToMarkdown('List managed devices', list_devices_hr, headers=HEADERS['device'])
+    return_outputs(human_readable, entry_context, raw_response)
+
+
+def get_managed_device_command(client: MsGraphClient, args: dict) -> None:
+    device_id: str = str(args.get('device_id'))
+    raw_response, device_id = client.get_managed_device(device_id)
+    device: dict = build_device_object(raw_response)
+    device_hr: dict = build_device_human_readable(raw_response)
+    entry_context: dict = {'MSGraphDeviceManagement.Device(val.ID === obj.ID)': device}
+    device_name: str = device.get('Name', '')
+    human_readable: str = f'Managed device {device_id} not found.'
+    if device:
+        human_readable = tableToMarkdown(f'Managed device {device_name}', device_hr, headers=HEADERS['device'])
```

Comment:
```suggestion

SPECIAL_HEADERS: dict =  {
    'raw_device': {
              'id': 'ID',
              'userId': 'User ID',
              'osVersion': 'OS Version',
              'imei': 'IMEI',
              'meid': 'MEID'
    }
}
HEADERS: dict = {
    'device': ['ID', 'User ID', 'Device Name', 'Operating System', 'OS Version', 'Email Address',
               'Manufacturer', 'Model', 'IMEI', 'MEID']
}


def convert_header_name(header: str) -> dict:
    """
   bla bla
    """
    return SPECIAL_HEADERS.get('raw_device').get(header, pascalToSpace(header))

        human_readable = tableToMarkdown(f'Managed device {device_name}', device_hr, headerTransform=convert_header_name, headers=HEADERS['device'], removeNull=True)
```
And now no need for `build_device_human_readable`